routes/web/project/location/postgres.rb

```diff
@@ -97,6 +97,13 @@ class CloverWeb
 
         r.redirect "#{@project.path}#{pg.path}"
       end
+
+      r.post "restart" do
+        Authorization.authorize(@current_user.id, "Postgres:create", @project.id)
+        Authorization.authorize(@current_user.id, "Postgres:edit", pg.id)
```

Comment:
Interesting, what's the thinking here? Is it really either of these, or something new?

Reply:
No, actually we don't need the first one. Removed it.